### PR TITLE
workloadentry handler is registered separately in serviceentry store

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: pilot/pkg/bootstrap/server.go [hunk 1]
```diff
@@ -918,6 +906,10 @@ func (s *Server) initRegistryEventHandlers() {
 				Resource().GroupVersionKind() {
 				continue
 			}
+			if schema.Resource().GroupVersionKind() == collections.IstioNetworkingV1Alpha3Workloadgroups.
```
Comment (ramaraochavali) [inline pilot/pkg/bootstrap/server.go:909]: Do you mean workloadEntry or workloadGroup?

Comment (hzxuzhonghu) [reply pilot/pkg/bootstrap/server.go:909]: It is workload group here

Comment (hzxuzhonghu) [reply pilot/pkg/bootstrap/server.go:909]: wlg currently can not influence any xds

**Review by ramaraochavali — APPROVED** [review]

**Pull request merged**